locations/spiders/q8_italia.py

```diff
@@ -0,0 +1,50 @@
+from scrapy import Spider
+
+from locations.items import Feature
+
+# Confusingly, not just IT but most of Europe
+# Also not just Q8Italia
+
+
+class Q8ItaliaSpider(Spider):
+    name = "q8_italia"
+    item_attributes = {"brand": "Q8", "brand_wikidata": "Q1634762"}
+    start_urls = ["https://www.q8.it/geolocalizzatore/pv/all"]
+
+    BRANDS = {
+        "TANGO": None,  # TangoSpider
```

Comment:
Tango wikidata is Q2423920

Reply:
Yeah but it's duplicate POIs from `tango.py` so I'm throwing them away on line 36. That way data consumers shouldn't have to do any de duplication across ATP data.